ssz/codec.py

```diff
@@ -0,0 +1,37 @@
+from functools import (
+    lru_cache,
+)
+
+from eth_utils import (
+    is_bytes,
+)
+
+from ssz.exceptions import (
+    DecodingError,
+)
+from ssz.utils import (
+    infer_sedes,
+)
+
+
+@lru_cache(128)
+def encode(obj):
+    '''
```

Comment:
Can we replace all `'''`s with `"""`s? This seems to be a convention in Ethereum's Python codebase, even if it isn't catched by the linter and I can't find a style guide requiring it.